components/tidb_query/src/rpn_expr/impl_cast.rs

```diff
@@ -779,6 +924,25 @@ mod tests {
         EvalContext::new(cfg)
     }
 
+    fn make_ctx_4(
```

Comment:
Could we apply a builder pattern here? Create a configuration structure to describe such flags.

Reply:
Different test need different flag, so if will create a config, it should had all flag as option. Then it will same as EvalContext.
This is test, and rust has no overload (if use trait, it is too troublesome). So it need to seperate these functions will `_x`